mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+			ARG 1 shiftX
+			ARG 2 shiftY
+			ARG 3 shiftZ
+			ARG 4 horizontalScale
+			ARG 6 verticalScale
+			ARG 8 noiseData
+			ARG 9 noise
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_eaemishu horizontalScale ()D
+		METHOD m_ebnaiicx createUnseeded (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;DDLnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_mdkmcezi$C_mnzzrhuq;
+			ARG 0 shiftX
+			ARG 1 shiftY
+			ARG 2 shiftZ
+			ARG 3 horizontalScale
+			ARG 5 verticalScale
+			ARG 7 noiseData
+		METHOD m_ectegnfr noise ()Lnet/minecraft/unmapped/C_rxrtfjrt;
+		METHOD m_etkmvjar shiftY ()Lnet/minecraft/unmapped/C_kzwapyud;
+		METHOD m_llovzvih verticalScale ()D
+		METHOD m_nbtngjbv shiftZ ()Lnet/minecraft/unmapped/C_kzwapyud;
+		METHOD m_swdjbmva nosieData ()Lnet/minecraft/unmapped/C_cjzoxshv;
+		METHOD m_uzggzava shiftX ()Lnet/minecraft/unmapped/C_kzwapyud;
+	CLASS C_neaittdk TransformerWithContext
+		METHOD m_ibixqdwp input ()Lnet/minecraft/unmapped/C_kzwapyud;
+		METHOD m_okdrvarc transform (Lnet/minecraft/unmapped/C_kzwapyud$C_rbrbrmhy;D)D
+			ARG 1 functionContext
+	CLASS C_onodflli Noise
+		FIELD f_guemmvit noise Lnet/minecraft/unmapped/C_rxrtfjrt;
+		FIELD f_jwvphzby DATA_CODEC Lcom/mojang/serialization/MapCodec;
+		FIELD f_mumgkjpp verticalScale D
+		FIELD f_wimimxkc horizontalScale D
+		FIELD f_zgzskmsv noiseData Lnet/minecraft/unmapped/C_cjzoxshv;
+		FIELD f_ziarbjkx CODEC Lcom/mojang/serialization/Codec;
+		METHOD <init> (Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_rxrtfjrt;DD)V
+			ARG 1 noiseData
+			ARG 2 noise
+			ARG 3 horizontalScale
+			ARG 5 verticalScale
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_blelxzrr noise ()Lnet/minecraft/unmapped/C_rxrtfjrt;
+		METHOD m_okqdqmzl createUnseeded (Lnet/minecraft/unmapped/C_cjzoxshv;DD)Lnet/minecraft/unmapped/C_mdkmcezi$C_onodflli;
+			ARG 0 noiseData
+			ARG 1 horizontalScale
+			ARG 3 verticalScale
+		METHOD m_oqyhwtko noiseData ()Lnet/minecraft/unmapped/C_cjzoxshv;
+		METHOD m_qmdpomly verticalScale ()D
+		METHOD m_vpcdieih horizontalScale ()D
+	CLASS C_oosxmyxk BlendOffset
+		FIELD f_zlflptle CODEC Lcom/mojang/serialization/Codec;
+	CLASS C_ozjfswew ShiftB
+		FIELD f_egmarlwv CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_jhnpsfqk offsetNoise Lnet/minecraft/unmapped/C_rxrtfjrt;
+		FIELD f_vbqqvgit noiseData Lnet/minecraft/unmapped/C_cjzoxshv;
+		METHOD <init> (Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_rxrtfjrt;)V
+			ARG 1 noiseData
+			ARG 2 offsetNoise
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+	CLASS C_pzgsokuu ShiftNoise
+		METHOD m_gbwiisgy withNewNoise (Lnet/minecraft/unmapped/C_rxrtfjrt;)Lnet/minecraft/unmapped/C_mdkmcezi$C_pzgsokuu;
+			ARG 1 offsetNoise
+		METHOD m_ggssjsxa compute (DDD)D
+			ARG 1 x
+			ARG 3 y
+			ARG 5 z
```

Comment:
Are these arg numbers correct?

Reply:
Doubles and longs use two lvt slots, so they increase the index by two instead of the usual one